pinot-core/src/test/java/org/apache/pinot/core/query/reduce/StreamingReduceServiceTest.java

```diff
@@ -78,19 +74,14 @@ public Void answer(InvocationOnMock invocationOnMock)
       }
     });
     final ExecutorService threadPoolService = Executors.newFixedThreadPool(1);
-    final ServerRoutingInstance routingInstance =
-        new ServerRoutingInstance("localhost", 9527, TableType.OFFLINE, false);
-    //We cannot use TestNG's annotation like @Test(expectedExceptions = { IOException.class }) to verify
+    final ServerRoutingInstance routingInstance = new ServerRoutingInstance("localhost", 9527, TableType.OFFLINE);
+    // We cannot use TestNG's annotation like @Test(expectedExceptions = { IOException.class }) to verify
     // because the Exception we hope to verify is nested inside the final exception.
     assertTrue(verifyException(() -> {
-          StreamingReduceService.processIterativeServerResponse(mock(StreamingReducer.class),
-              threadPoolService,
-              ImmutableMap.of(routingInstance, mockedResponse),
-              10,
-              mock(BaseReduceService.ExecutionStatsAggregator.class));
-          return null;
-        },
-        (cause) -> cause instanceof TimeoutException));
+      StreamingReduceService.processIterativeServerResponse(mock(StreamingReducer.class), threadPoolService,
+          ImmutableMap.of(routingInstance, mockedResponse), 10, mock(BaseReduceService.ExecutionStatsAggregator.class));
+      return null;
+    }, (cause) -> cause instanceof TimeoutException));
```

Comment:
Does this test actually guarantee the callback executes?

Reply:
I didn't check the test logic, and this test is actually unrelated to the PR except for the `ServerRoutingInstance` constructor change. Reverted the unnecessary reformat change to reduce the noise for the review